LobbyServer2/ApiServer/UserApiServer.cs

```diff
@@ -31,6 +33,7 @@ protected override void ConfigureApp(WebApplication app)
         app.MapPost("/api/login", Login).AllowAnonymous();
         app.MapPost("/api/register", Register).AllowAnonymous();
         app.MapGet("/api/lobby/status", StatusController.GetSimpleStatus).AllowAnonymous();
+        app.MapGet("/api/lobby/getPlayerInfo", GetPlayerInfo).RequireAuthorization();
```

Comment:
a nitpick but you already have GET as a method (MapGet), get in endpoint name is kind of redundant and not RESTful

Reply:
C# method name could have stayed as it was but whatever